`<use_ />` does not work in view! macro

**Describe the bug**
I am using the `<use_ />` component for a project that I am rewriting in leptos. I first tried just `<use />` which did not work as it is a keyword, and then I saw that you had renamed it to use_. However when I tried it, I got the following error

```bash
   Compiling my_package v0.1.0 (/my_project/my_package)
error[E0433]: failed to resolve: could not find `Use_` in `svg`
  --> my_package/src/my_component.rs:39:33
   |
39 | ...   view! { cx, <use_ href="#cell" x=0 y=0 /> }
   |       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |       |
   |       could not find `Use_` in `svg`
   |       help: a struct with a similar name exists: `Use`
   |
   = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)

warning: unused import: `svg::use_`
 --> my_package/src/my_component.rs:1:17
  |
1 | use leptos::{*, svg::use_};
  |                 ^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default

For more information about this error, try `rustc --explain E0433`.
```

**Leptos Dependencies**

```toml
leptos = { version = "0.4", features = ["nightly"] }
leptos_meta = { version = "0.4", features = ["nightly"] }
leptos_router = { version = "0.4", features = ["nightly"] }
leptos_axum = { version = "0.4" }
```

**To Reproduce**
A simple case would be (from mdn)
```html
<svg viewBox="0 0 30 10" xmlns="http://www.w3.org/2000/svg">
  <circle id="myCircle" cx="5" cy="5" r="4" stroke="blue" />
  <use_ href="#myCircle" x="10" fill="blue" />
  <use_ href="#myCircle" x="20" fill="white" stroke="red" />
</svg>
```

**Expected behavior**
I would expect either the `<use />` or `<use_ />` component to build without error.

--- a/leptos_macro/src/view.rs
+++ b/leptos_macro/src/view.rs
@@ -551,7 +551,7 @@ fn element_to_tokens_ssr(
             }
 
             template.push_str("</");
-            template.push_str(&node.name().to_string());
+            template.push_str(tag_name);
             template.push('>');
         }
     }

